### PR TITLE
Process as platform specific files only these located in app folder

### PR DESCRIPTION
**Pull request by Fatme**
https://github.com/NativeScript/nativescript-cli/issues/514

### CODE REVIEW

File: lib/services/platform-service.ts [hunk 1]
```diff
@@ -157,15 +157,16 @@ export class PlatformService implements IPlatformService {
 			}
 
 			// Process platform specific files
-			var contents = this.$fs.readDirectory(platformData.appDestinationDirectoryPath).wait();
+			var contents = this.$fs.readDirectory(path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME)).wait();
```
Comment (teobugslayer) [inline lib/services/platform-service.ts]: let :)

**Comment by teobugslayer** [comment]
:+1: after addressing my small remark.

File: lib/services/platform-service.ts (hunk 1 above)
Comment (Fatme) [inline lib/services/platform-service.ts]: I'm planning to replace all `var` usages with `let` in another PR.

**Comment by ErjanGavalji** [comment]
:+1:

**Pull request merged**